ci/docker/x86_64-musl/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ubuntu:16.04
+
+RUN apt-get update -y && apt-get install -y gcc gcc-aarch64-linux-gnu ca-certificates git
+RUN git config --global --add safe.directory '*'
+
+ENV PATH=$PATH:/rust/bin
+ENV CARGO_BUILD_TARGET=aarch64-unknown-linux-gnu
+ENV CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER=aarch64-linux-gnu-gcc
```

Comment:
I think there may have been a mixup maybe? These variables look like they should be in the above dockerfile, and down here I think the `-crt-static` is needed.

Reply:
I did the classic not saving the file :') will commit the correct one